roc/version.go

```diff
@@ -84,8 +84,12 @@ func (vi VersionInfo) Validate() error {
 }
 
 func checkVersion() {
+	var vi VersionInfo
+	logWrite(LogDebug, "entering versionCheck()")
+	defer logWrite(LogDebug, "leaving versionCheck(): version=%+v", vi)
```

Comment:
There is no need to log every checkVersion call because in all invocations except first it's no-op.

Instead, it would be enough to log the retrieved version only once, after fetchVersion call and before validating it.